spec.html

```diff
@@ -25624,17 +25636,15 @@ <h1>
         1. Set the VariableEnvironment of _scriptContext_ to _globalEnv_.
         1. Set the LexicalEnvironment of _scriptContext_ to _globalEnv_.
         1. Set the PrivateEnvironment of _scriptContext_ to *null*.
-        1. Suspend the running execution context.
         1. Push _scriptContext_ onto the execution context stack; _scriptContext_ is now the running execution context.
         1. Let _script_ be _scriptRecord_.[[ECMAScriptCode]].
         1. Let _result_ be Completion(GlobalDeclarationInstantiation(_script_, _globalEnv_)).
         1. If _result_.[[Type]] is ~normal~, then
           1. Set _result_ to Completion(Evaluation of _script_).
           1. If _result_.[[Type]] is ~normal~ and _result_.[[Value]] is ~empty~, then
             1. Set _result_ to NormalCompletion(*undefined*).
-        1. Suspend _scriptContext_ and remove it from the execution context stack.
+        1. Remove _scriptContext_ from the execution context stack and restore the execution context that is at the top of the execution context stack as the running execution context.
         1. Assert: The execution context stack is not empty.
```

Comment:
```suggestion
        1. Remove _scriptContext_ from the execution context stack.
        1. Assert: The execution context stack is not empty.
        1. NOTE: The execution context that is at the top of the execution context stack is now the running execution context.
```

We don't *make* something the running execution context. By the definition of the term, it just *is* because of its position on the top of the stack.

Reply:
I think your point is that the wording "and restore ... as the running execution context" misleadingly suggests the need to *do* something (extra) to make that happen. I basically agree.

However, the "Remove ... and restore ..." wording is well-established in the status quo, in lots of places that this PR doesn't touch, so fixing it should probably be the job of a different PR.